### PR TITLE
fix(avatarGroup): replace style tag with classname in stories

### PR DESCRIPTION
**Pull request by Rashi-Gupta-2000**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/avatarGroup/__stories__/variants/style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.Avatar-group {
+    max-height: 150x;
```
Comment (satyamyadav) [inline core/components/atoms/avatarGroup/__stories__/variants/style.css]: Please use tokens like : `max-height: var(--spacing-7); `

File: core/components/atoms/avatarGroup/__stories__/variants/CustomPopper.story.jsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { AvatarGroup, Avatar, Text } from '@/index';
 import { list } from '../AvatarList';
+import './style.css';
```
Comment (satyamyadav) [inline core/components/atoms/avatarGroup/__stories__/variants/CustomPopper.story.jsx:4]: Please add a way to communicate the user to include this style in code.

File: core/components/atoms/avatarGroup/__stories__/variants/CustomPopper.story.jsx [hunk 3]
```diff
@@ -82,7 +79,7 @@ const customCode = `() => {
     });
 
     return (
-      <div className="overflow-auto py-4 px-6" style={{ maxHeight: '150px' }}>
+      <div className="overflow-auto py-4 px-6 Avatar-group">
```
Comment (satyamyadav) [inline core/components/atoms/avatarGroup/__stories__/variants/CustomPopper.story.jsx]: Please use some specific class name, `Avatar-group` is too generic and can conflict. The name can be like "UserAvatars-popover".

**Review by satyamyadav — CHANGES_REQUESTED** [review]

File: core/components/atoms/avatarGroup/__stories__/variants/CustomPopper.story.jsx [hunk 4]
```diff
@@ -21,17 +22,19 @@ export const customPopper = () => {
...
-    );
+    return <div className="overflow-auto py-4 px-6 UserAvatars-popover">{AvatarList}</div>;
   };
 
   return <AvatarGroup list={list} popoverOptions={{ popperRenderer, dark: false, on: 'click' }} />;
 };
 
-const customCode = `() => {
+const customCode = `/*
+.UserAvatars-popover {
```
Comment (satyamyadav) [inline core/components/atoms/avatarGroup/__stories__/variants/CustomPopper.story.jsx:33]: Please add a CSS file name suggestion to communicate this needs to be written in a file.

**Review by satyamyadav — APPROVED** [review]

File: core/components/atoms/avatarGroup/__stories__/variants/CustomPopper.story.jsx (hunk 4 above)
Comment (Rashi-Gupta-2000) [reply core/components/atoms/avatarGroup/__stories__/variants/CustomPopper.story.jsx:33]: added

**Pull request merged**